src/Observable.ts

```diff
@@ -141,7 +141,7 @@ export class Observable<T> implements Subscribable<T> {
         } else {
           // if there is NO subscription, then we're getting a nexted
           // value synchronously during subscription. We can just call it.
-          // If it errors, Observable's `subscribe` imple will ensure the
```

Comment:
btw I believe this was intended to be 

> Observable's subscribe _simply_ will ensure the

but it reads better without it, so 👍